### DISCUSSION TITLE
steps output

### DISCUSSION THREAD

**Discussion by rectalogic** [community/community#26108] [Actions]
<p>I run a step that sets an output, but the steps object is empty in the next step.</p>
<p>This is my test workflow <a href="https://github.com/rectalogic/test-actions/blob/master/.github/workflows/steps.yml" rel="nofollow noopener">https://github.com/rectalogic/test-actions/blob/master/.github/workflows/steps.yml</a></p>
<p>And this is the output:</p>
<p><a href="https://github.com/rectalogic/test-actions/commit/5337662676d1bbaa710ae9829386dbc4474fafb1/checks" rel="nofollow noopener">https://github.com/rectalogic/test-actions/commit/5337662676d1bbaa710ae9829386dbc4474fafb1/checks</a></p>
<p>I can see the github-script is setting the output:</p>
<p><span><span>##[set-output name=result;]“Hello world”</span></span></p>

**Comment by perseus-me** [comment, marked as answer]
<p>Add <code>id</code> field to you step, see <a href="https://github.com/rusnasonov/vscode-hugo/blob/master/.github/workflows/build.yml" rel="nofollow noopener">https://github.com/rusnasonov/vscode-hugo/blob/master/.github/workflows/build.yml</a></p>
